optimum/onnxruntime/modeling_ort.py

```diff
@@ -524,6 +524,7 @@ def prepare_io_binding(
             batch_size=input_ids.size(0),
```

Comment:
Good catch, can you also update the comment from
`# bind logits` to `# bindlast_hidden_state` ?

Reply:
Thanks!